src/backends/sentry_backend_crashpad.cpp

```diff
@@ -141,7 +146,9 @@ sentry__crashpad_handler(int signum, siginfo_t *info, ucontext_t *user_context)
     sentry_value_t event = sentry_value_new_event();
 
     SENTRY_WITH_OPTIONS (options) {
-
+        auto *data = static_cast<crashpad_state_t *>(options->backend->data);
+        data->event_id = sentry_value_get_by_key(event, "event_id");
```

Comment:
I guess there should never be the situation where `data->event_id` has already been set / needs to be freed first right?